docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -94,6 +94,23 @@ Example:
 serviceAccount: "weblogic-operator"
 ```
 
+##### `dedicated`
+
+Specifies if this operator will only manage WebLogic Domains in the same namespace in which the operator itself is deployed.  If set as `true`, then the `domainNamespaces` value below is ignored. 
+
+Defaults to `false`.
+
+Example:
+```
+dedicated: false
+```
+
+In the `dedicated` mode, the operator will most likely not have permissions to access the cluster-scoped Kubernetes resources such as CustomResourceDefinitions, PersistentVolumes, and Namespaces. In those situations, the operator may skip some of its operations, such as verifying the WebLogic Domain CustomResoruceDefinition domains.weblogic.oracle (and creating it when it is absent), watching namespace events, and cleaning up PersistentVolumes as part of deleting a domain. 
```

Comment:
don't say "will most likely not have"  
say "does not require"

Reply:
backtick things like CustomResourceDefinitions

Reply:
done